gpu-operator/release-notes.rst

```diff
@@ -34,6 +34,93 @@ See the :ref:`GPU Operator Component Matrix` for a list of software components a
 
 ----
 
+.. _v24.9.0:
+
+24.9.0
+======
+
+.. _v24.9.0-new-features:
+
+New Features
+------------
+
+* Added support for the NVIDIA Data Center GPU Driver version 550.127.05.
```

Comment:
Do we also mention the `535.216.01` and `565.57.01` driver versions here?

Reply:
My memory is that we identify the new default LTSB driver and the others are just post-release updates.